pkg/kn/commands/service/human_readable_flags.go

```diff
@@ -30,7 +30,7 @@ func ServiceListHandlers(h hprinters.PrintHandler) {
 		{Name: "Namespace", Type: "string", Description: "Namespace of the Knative service", Priority: 0},
 		{Name: "Name", Type: "string", Description: "Name of the Knative service.", Priority: 1},
 		{Name: "Url", Type: "string", Description: "URL of the Knative service.", Priority: 1},
-		{Name: "Generation", Type: "integer", Description: "Sequence number of 'Generation' of the service that was last processed by the controller.", Priority: 1},
+		{Name: "Latest Revision", Type: "string", Description: "Name of the latest revision.", Priority: 1},
```

Comment:
Should we have space in column names? Can we change the column name to `Latest` as suggested in review?
Also per changes, the description should read, "Name of the latest ready revision."

Reply:
Changed to `Latest`. The comment is changed to `Name of the latest ready revision.`.